schematic/store/synapse.py

```diff
@@ -1706,6 +1704,7 @@ def upload_manifest_as_table(
         table_manipulation: str,
         table_column_names: str,
         annotation_keys: str,
+        file_annotations_upload: Optional[bool] = True,
```

Comment:
Does this need to be optional? It seems that True/False is all thats needed here

Reply:
I see what you meant. Yes, i could update this to just `file_annotations_upload: bool = True,`